draft-dr-dprive-tls-and-dtls-authentication.txt

```diff
@@ -141,91 +149,86 @@ Internet-Draft            (D)TLS Authentication            November 2015
    "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this
    document are to be interpreted as described in [RFC2119].
 
-   We use several terms specifically in the context of this draft:
+   Several terms are used specifically in the context of this draft:
 
-   o  DNS Client: a DNS stub implementation or forwarding resolver.  A
-      DNS Client receives requests from an application or a system and
-      forwards them to some pre-configured set of upstream DNS
-      Resolvers.  In the case of a forwarding resolver, we use the term
-      "DNS Client" to discuss its active or querying side here.
+   o  DNS client: a DNS stub resolver or forwarder/proxy.  In the case
+      of a forwarder, the term "DNS client" is used to discuss the side
+      that sends queries.
 
-   o  DNS Resolver: a pre-configured upstream resolver that a DNS client
-      talks to.  These are usually recursive resolvers, but may
-      themselves also be forwarding resolvers.  In the case of a
-      forwarding resolver, we use the term "DNS Resolver" to discuss its
-      passive or responding side here.
+   o  DNS server: a DNS recursive resolver or forwarder/proxy.  In the
+      case of a forwarder, the term "DNS server" is used to discuss the
+      side that responds to queries.
 
-   o  Reference Identifier: a Reference Identifier as described in
-      [RFC6125], constructed by the DNS client when connecting to the
-      DNS server.
+   o  Credential: Information available for a DNS server which proves
+      its identity.  Credentials discussed here include:
 
-   o  Credential: Information presented by a DNS Resolver during TLS
-      authentication which proves its identity.  Covered credentials
-      include:
 
 
 
-Dickinson & Reddy         Expires May 19, 2016                  [Page 3]
+Dickinson & Reddy         Expires May 21, 2016                  [Page 3]
 
 Internet-Draft            (D)TLS Authentication            November 2015
 
 
       *  X.509 certificate
 
-      *  DNSSEC chain to TLSA record
+      *  DNSSEC validated chain to a TLSA record
 
    o  SPKI Pinsets: [I-D.ietf-dprive-dns-over-tls] describes the use of
       cryptographic digests to "pin" public key information in a manner
       similar to HPKP [RFC7469].  An SPKI pinset is a collection of
-      these pins that constrains a DNS Resolver.
+      these pins that constrains a DNS server.
+
+   o  Reference Identifier: a Reference Identifier as described in
+      [RFC6125], constructed by the DNS client when performing TLS
+      authentication of a DNS server.
 
    o  (D)TLS: Transport Layer Security [RFC5246] and (possibly) Datagram
       Transport Layer Security [RFC6347]
 
-   o  Privacy-protective DNS Server: A DNS Resolver that:
+   o  Privacy-enabling DNS server: A DNS server that:
 
```

Comment:
I changed this to privacy-enabling since I thought privacy-protective might be taken to imply it ONLY does (D)TLS, where as in practice (I think) most servers will do TCP/UDP too. But this does raise an interesting question about that!